replace "analysis" with "asset" in asset-paths doc (#2371)
fishtown-analytics/docs.getdbt.com: website/docs/reference/project-configs/asset-paths.md

</Changelog>

## Default

> By default, dbt will not compile any additional files as part of `docs generate`, i.e. `asset-paths: []`

Reviewer: thanks for opening this up @kopackiw! i believe what the docs is saying is that your analyses (saved in `analysis-path`) will not be generated by a `dbt docs generate`. I've suggested the below to clarify a bit further:

```suggestion
By default, dbt will not copy any additional files as part of docs generate (for example `asset-paths: []`). 
```

Reviewer: hey @b-per does this look right to you?

Reviewer 2: I think that the issue that is being reported is that the `Default` section of the `asset-paths` page talks about `analysis-paths`.

I'd change it to: 
By default, dbt will not copy any additional files as part of `docs generate`, i.e. `asset-paths: []`

Replacing `copy` by `compile` and using the correct config item.

> By default, dbt will not copy any additional files as part of docs generate (for example `asset-paths: []`). 

Reviewer: ```suggestion
By default, dbt will not copy any additional files as part of docs generate, i.e. `asset-paths: []`
```
